package.json

```diff
@@ -15,6 +15,7 @@
     "mocha-webpack": "./node_modules/.bin/mocha-webpack --webpack-config build/webpack.test.conf.js --require test/setup.ts"
   },
   "dependencies": {
+    "axios": "^0.17.1",
```

Comment:
If the library does not come with Typescript definitions we should be downloading them at `@types/axios`. Disregard if it does have types already

Reply:
Yup types included